howto.md

```diff
@@ -67,6 +67,19 @@ Writing example:
 
 ```
 
+## How to access S3 anonymously
+
+```python
+>>> import botocore
+>>> import botocore.client
+>>> from smart_open import open
+>>> config = botocore.client.Config(signature_version=botocore.UNSIGNED)
+>>> params = {'resource_kwargs': {'config': config}}
+>>> with open('s3://commoncrawl/robots.txt', transport_params=params) as fin:
+...    fin.readline()
+'User-Agent: *\n'
```

Comment:
As opposed to what?

Can you add a motivational sentence under the heading title?